include/adc/eps.h

```diff
@@ -1,32 +1,37 @@
+#ifndef ADC_EPS_H
```

Comment:
It's unlikely that these header files will be included multiple times; it might actually be useful to *not* have header guards here, so if this is happening, we can figure out why. Running into a multiple defines problem with this kind of file probably means something is wrong with the code architecture.

Reply:
@brunofalmeida